code/modules/admin/admin_verbs.dm

```diff
@@ -228,19 +228,15 @@ var/list/admin_verbs_mentor = list(
 	set name = "Invisimin"
 	set category = "Admin"
 	set desc = "Toggles ghost-like invisibility (Don't abuse this)"
+
 	if(holder && mob)
 		if(mob.invisibility == INVISIBILITY_OBSERVER)
 			mob.invisibility = initial(mob.invisibility)
-			mob << "\red <b>Invisimin off. Invisibility reset.</b>"
-			mob.icon_state = "ghost"
-			mob.icon = 'icons/mob/human.dmi'
-			mob.update_icons()
+			mob << "<span class='danger'>Invisimin off. Invisibility reset.</span>"
+			//TODO: Make some kind of indication for the badmin that they are currently invisible

```

Comment:
@Zuhayr Eehh, I'm iffy on doing that, primarily because invisimin is generally used for events, and I know at least I tinker with alpha a lot.